torch/nn/modules/linear.py

```diff
@@ -42,6 +42,7 @@ class Linear(Module):
         >>> print(output.size())
         torch.Size([128, 30])
     """
+    __constants__ = ['bias']
```

Comment:
Uhhh what? Should we really treat it as a constant when it’s a valid parameter?

Reply:
When it's a valid parameter, we don't look for the constant list, only when it's a NoneType, we look for this constant list and treat NoneType as constant